src/login/oidc/oidcHandlers/LegacyImplicitFlowOidcHandler.ts

```diff
@@ -49,33 +48,35 @@ export default class LegacyImplicitFlowOidcHandler implements IOidcHandler {
     );
   }
 
-  async handle(oidcLoginOptions: IOidcOptions): Promise<ISolidSession> {
-    const requestUrl = new URL(
-      oidcLoginOptions.issuerConfiguration.authorizationEndpoint.toString()
-    );
-    // TODO: include client_id, state, and nonce
-    // Disable camel case rule because this query requires camel case
-    /* eslint-disable @typescript-eslint/camelcase */
-    const query: { [key: string]: string } = {
-      response_type: "id_token token",
-      redirect_url: oidcLoginOptions.redirectUrl.toString(),
-      scope: "openid id_vc"
-    };
-    /* eslint-enable @typescript-eslint/camelcase */
-    if (oidcLoginOptions.dpop) {
-      query.dpop = await this.dpopHeaderCreator.createHeaderToken(
-        oidcLoginOptions.issuer,
-        "GET"
-      );
-    }
-    requestUrl.set("query", query);
+  async handle(oidcLoginOptions: IOidcOptions): Promise<ISessionInfo> {
+    throw new Error("Not Implemented");
+
+    // const requestUrl = new URL(
```

Comment:
Should this commented code be removed?

Reply:
Fixed in 797d8f257190c0f8687c86c7595580ae48509a01